maps/random_ruins/exoplanet_ruins/playablecolony/playablecolony.dm

```diff
@@ -21,6 +21,23 @@
 	info = "You are a Colonist, living on the rim of explored, let alone inhabited, space in a reconstructed shelter made from the very ship that took you here."
 	total_positions = 6
 	outfit_type = /decl/hierarchy/outfit/job/colonist
+	min_skill = list(
+		SKILL_LITERACY = SKILL_BASIC,
+		SKILL_CONSTRUCTION = SKILL_BASIC,
+		SKILL_ANATOMY = SKILL_BASIC,
```

Comment:
maybe bit too generous on medical skills, MAYBE medical basic if you want everyone to be paramedic tier, but anatomy training for miners is weird.